<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>477</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>477</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>706</width>
    <height>477</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Desafio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/networking (1).png</normaloff>../../Downloads/networking (1).png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 229, 229);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>29</y>
      <width>631</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxSalas">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
</string>
       </property>
       <property name="title">
        <string>Salas</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>411</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="salasBtnCadastrar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(103, 154, 0);</string>
           </property>
           <property name="text">
            <string>Cadastrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="salasBtnConsultar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="text">
            <string>Consultar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBoxSalasCafe">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="title">
        <string>Salas de Caf√©</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>19</y>
          <width>411</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="salasCafeBtnCadastrar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(103, 154, 0);</string>
           </property>
           <property name="text">
            <string>Cadastrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="salasCafeBtnConsultar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="text">
            <string>Consultar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBoxPessoas">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="title">
        <string>Pessoas</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>20</y>
          <width>411</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pessoasBtnCadastrar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(103, 154, 0);</string>
           </property>
           <property name="text">
            <string>Cadastrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pessoasBtnConsultar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="text">
            <string>Consultar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCadastrar">
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionConsultar">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
  <action name="actionCadastrar_2">
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionConsultar_2">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
  <action name="actionCadastrar_3">
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionConsutlar">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
